lib/ferto/client.rb

```diff
@@ -153,6 +160,19 @@ def build_body(aggr_id, aggr_limit, url, callback_url, callback_type,
         body[:extra] = extra.is_a?(Hash) ? extra.to_json : extra.to_s
       end
 
+      # We will continue to expose the user_agent field just like tools
+      # like curl and wget do. Along with that we will follow their paradigm
+      # where if both a user-agent flag and a `User-Agent` in the request headers
+      # are provided then the user agent in the request headers is preferred.
+      #
+      # Also if the `user_agent` is provided but the request headers do not
+      # contain a `User-Agent` key, then the `user_agent` is copied to the headers
```

Comment:
I think this should be also documented in the README. In case a user sets both, it would be confusing that the one he sets specifically using `user_agent `, it will not be respected and overwritten by the header's one. The priority that we follow should be clear to the user.

Reply:
Right! The decision is documented in README now.